### DISCUSSION TITLE
any rules to remember spacing numbers?

### DISCUSSION THREAD

**Discussion by videni** [tailwindlabs/tailwindcss#6866] [Help]
hi, I am new to tailwindcss, any way to remember spacing numbers?  for example px-4(1rem), px-1 (0.25rem),  max-w-7xl(80 rem)?    is there a quick way to remember these number please?

**Comment by adamwathan** [comment]
For the numeric spacing scales, one "Tailwind unit" is 4px or 0.25rem. So pt-1 is 0.25rem (4px), pt-2 is 0.5rem (8px), and pt-10 is 2.5rem (40px).

For the t-shirt scales like `max-w-7xl` there's not really a mathematical system, the intended workflow is that you just sort of try out the values until you find one that looks good to you. If you are trying to match a design given to you by a designer, I would leverage the IntelliSense plugin which will show you the values:

![image](https://user-images.githubusercontent.com/4323180/148058072-d8e3531b-5356-4c1e-ba07-89f4a1c7f040.png)
